server/mediaserver.go

```diff
@@ -358,10 +360,10 @@ func (s *LivepeerServer) registerConnection(rtmpStrm stream.RTMPVideoStream) (*r
 	}
 	storage := drivers.NodeStorage.NewSession(string(mid))
 	// Build the source video profile from the RTMP stream.
-	resolution := fmt.Sprintf("%vx%v", rtmpStrm.Width(), rtmpStrm.Height())
+	// Build the source video profile from the RTMP stream.
```

Comment:
I think this will break with RTMP ingest

Reply:
What part? I'm testing locally and things seem to work fine.

Reply:
Is the resolution in the master playlist actually populated with RTMP?

Reply:
I'll check. Is the resolution we're looking for populated here?
```golang
playlist := core.NewBasicPlaylistManager(mid, storage)
masterPL := playlist.GetHLSMasterPlaylist()
variants := masterPL.Variants
for i, v := range variants {
       resolution := v.Resolution
       glog.Error("resolution: ", resolution)
}
```

Reply:
Yeah, or `curl http://localhost:8935/stream/mid.m3u8` to see the resolution as part of the m3u8 header while you're streaming.

Reply:
I've added the resolution back like this and all works:
``` golang
if params.resolution == "" {
       params.resolution = fmt.Sprintf("%vx%v", rtmpStrm.Width(), rtmpStrm.Height())
}
```

Reply:
Yep. Thought we would have access to the `rtmpStrm` from within the `createStreamID` callback but that's not the case, so this line is needed. 👍